src/utils.c

```diff
@@ -2414,17 +2419,39 @@ void utils_start_new_geany_instance(const gchar *doc_path)
 }
 
 
+#if defined(G_OS_WIN32) && !defined(HAVE_REALPATH)
+/* realpath implementation for Windows found at http://bugzilla.gnome.org/show_bug.cgi?id=342926
+ * this one is better than e.g. liberty's lrealpath because this one uses Win32 API and works
+ * with special chars within the filename */
+static char *realpath(const char *pathname, char *resolved_path)
+{
+	int size;
+
+	if (resolved_path != NULL)
+	{
+		size = GetFullPathName(pathname, PATH_MAX, resolved_path, NULL);
```

Comment:
Previously the ANSI version was called explicitly. What was your reason to change that?

Reply:
I had read that this version of the function would automatically use the A or W version depending on whether unicode is enabled.  But further reading indicates that it's not so straightforward to switch between the two versions.  So will revert the change.